### PR TITLE
Add org.pencil2d.Pencil2D

### PR DESCRIPTION
**Pull request by eszlari**

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build org.pencil2d.Pencil2D

**Comment by eszlari** [comment]
appdata PR: https://github.com/pencil2d/pencil/pull/1035

File: org.pencil2d.Pencil2D.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+command: pencil2d
+rename-desktop-file: pencil2d.desktop
+rename-icon: pencil2d
+finish-args:
+  - --socket=wayland
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host # to make saving work
```
Comment (barthalion) [inline org.pencil2d.Pencil2D.yaml]: It needs entire host access for saving files?

**Comment by barthalion** [comment]
Hi @Jose-Moreno, should entire core team get push access to the new Flathub repo?

**Comment by Jose-Moreno** [comment]
@barthalion Hello! Thank you for contacting me. I'm not a developer myself as I'm in charge of user support and communications within the Pencil2d team, but I think it's safe to provide push access for most of the current core team. These are the main developers username handles:
- @chchwy
- @CandyFace
- @scribblemaniac
- @j5lx

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.pencil2d.Pencil2D

**Comment by nedrichards** [comment]
I've sent invitations to all the requested usernames.

**Pull request closed without merging**